Including "rc1" in filenames for UI plugin is unacceptable to ExtensionManager

@dbarkelew and I have been trying to get the 1.1.1 plugins to deploy, without success. It seems that including `rc1` in the filenames for the packages that are posted on https://ova_address:9443 causes ExtensionManager not to download the packages. The plugin registration succeeds, but nothing is visible in the UI because the package was never imported into VC. This makes it impossible to test the UI deployment with a non-GA build.

Here's an excerpt from the virgo log:

```
[2017-05-04T16:14:58.871Z] [ERROR] vc-extensionmanager-pool-227 70000086 100023 200021 com.vmware.vise.extensionfw.ExtensionManager                      Error retrieving deployment data for the plugin package at /etc/vmware/vsphere-client/vc-packages/vsphere-client-serenity/com.vmware.vic-1.1.1-rc1 java.io.IOException: Invalid plugin package version for com.vmware.vic
 Format should be 'major.minor.micro.build' (i.e. 4, 4.5, 4.5.1 or 4.5.1.123). Got this instead: 1.1.1-rc1
        at com.vmware.vise.extensionfw.impl.PackageManifestParser.parse(PackageManifestParser.java:114)
        at com.vmware.vise.extensionfw.ExtensionManager.getPackageDeployData(ExtensionManager.java:1444)
        at com.vmware.vise.extensionfw.ExtensionManager.addPluginPackages(ExtensionManager.java:461)
        at com.vmware.vise.extensionfw.impl.ExtensionServiceImpl.addPluginPackages(ExtensionServiceImpl.java:171)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:319)
        at org.eclipse.gemini.blueprint.service.importer.support.internal.aop.ServiceInvoker.doInvoke(ServiceInvoker.java:56)
        at org.eclipse.gemini.blueprint.service.importer.support.internal.aop.ServiceInvoker.invoke(ServiceInvoker.java:60)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
        at org.springframework.aop.support.DelegatingIntroductionInterceptor.doProceed(DelegatingIntroductionInterceptor.java:131)
        at org.springframework.aop.support.DelegatingIntroductionInterceptor.invoke(DelegatingIntroductionInterceptor.java:119)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
        at org.eclipse.gemini.blueprint.service.importer.support.LocalBundleContextAdvice.invoke(LocalBundleContextAdvice.java:57)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
        at org.springframework.aop.support.DelegatingIntroductionInterceptor.doProceed(DelegatingIntroductionInterceptor.java:131)
        at org.springframework.aop.support.DelegatingIntroductionInterceptor.invoke(DelegatingIntroductionInterceptor.java:119)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
        at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:202)
        at com.sun.proxy.$Proxy246.addPluginPackages(Unknown Source)
        at com.vmware.vise.vim.extension.VcExtensionManager$2.call(VcExtensionManager.java:710)
        at com.vmware.vise.vim.extension.VcExtensionManager$2.call(VcExtensionManager.java:695)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at com.vmware.vise.util.concurrent.QueuingCachedThreadPool$QueueProcessor.run(QueuingCachedThreadPool.java:885)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:745)
```

--- a/Makefile
+++ b/Makefile
@@ -411,8 +411,8 @@ ENV_HTML_SDK_HOME = "/tmp/sdk/html-client-sdk"
 
 vic-ui-plugins:
 	@npm install -g yarn > /dev/null
-	sed "s/0.0.1/$(shell printf %s ${TAG_NUM})/" ./$(VICUI_SOURCE_PATH)/plugin-package.xml > ./$(VICUI_SOURCE_PATH)/new_plugin-package.xml
-	sed "s/0.0.1/$(shell printf %s ${TAG_NUM})/" ./$(VICUI_H5_UI_PATH)/plugin-package.xml > ./$(VICUI_H5_UI_PATH)/new_plugin-package.xml
+	sed -e "s/0.0.1/$(shell printf %s ${TAG_NUM})/" -e "s/\-rc[[:digit:]]//g" ./$(VICUI_SOURCE_PATH)/plugin-package.xml > ./$(VICUI_SOURCE_PATH)/new_plugin-package.xml
+	sed -e "s/0.0.1/$(shell printf %s ${TAG_NUM})/" -e "s/\-rc[[:digit:]]//g" ./$(VICUI_H5_UI_PATH)/plugin-package.xml > ./$(VICUI_H5_UI_PATH)/new_plugin-package.xml
 	sed "s/UI_VERSION_PLACEHOLDER/$(shell printf %s ${TAG})/" ./$(VICUI_H5_SERVICE_PATH)/src/main/resources/configs.properties > ./$(VICUI_H5_SERVICE_PATH)/src/main/resources/new_configs.properties
 	rm ./$(VICUI_SOURCE_PATH)/plugin-package.xml ./$(VICUI_H5_UI_PATH)/plugin-package.xml ./$(VICUI_H5_SERVICE_PATH)/src/main/resources/configs.properties
 	mv ./$(VICUI_SOURCE_PATH)/new_plugin-package.xml ./$(VICUI_SOURCE_PATH)/plugin-package.xml

